canonical/charmhub.io
# Accept and reject invites

## Done
Added ability to accept and reject invites

## How to QA
- In a private browser create a new account (e.g. firstname.lastname+testing@canonical.com)
- In your regular account go to https://charmhub-io-1747.demos.haus/<CHARM_NAME>/collaboration
- Invite the new account as a collaborator
- You will see the following error: "Error: There is already an invite for this email address"
- Ignore the error as it will be addressed separately
- In the notification that appears after the invite is created, copy the link address of the "Accept invite" link
- Paste it in a private browser and login as your new account
- Once redirected you'll need to paste the link in again as the redirect loses the token
- Accept invite
- Go to https://charmhub-io-1747.demos.haus/<CHARM_NAME>/listing with your new account
- If it's worked then you should be able to access that page

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-8221
